Update docker repo address

jdknives opened the pull request:
Update all references to SkycoinPro since we updated the paths and repo addresses.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/skycoin/skycoin-service-discovery?pullRequest=5) <br/>All committers have signed the CLA.

jdknives committed:
Fix docker repo addresses

Since we moved this repo and dockerhub repo to `skycoin` org, we need
to update docker push paths and tags and documenation.
